internal/buildscripts/update-deps

```diff
@@ -40,30 +42,47 @@ for gomod in $( find "$REPO_DIR" -name "go.mod" | grep -v "/examples/" | sort );
         else
             pkg="$( echo "$line" | awk '{print $1}' )"
         fi
+        echo go mod edit -replace="${pkg}"="${pkg}"@$(desired_contrib_version "$line")
         go mod edit -replace="${pkg}"="${pkg}"@$(desired_contrib_version "$line")
-        go mod tidy
+#        go mod tidy
```

Comment:
I've yet to finish updating this script but we shouldn't attempt to tidy here because it's all but guaranteed to fail after bumping a single component dep and none of the others, including core.

Reply:
:+1: and we can disregard failure of that build for this PR.